### PR TITLE
Ignore expected swss, syncd error messages for J2C+ duts during autorestart feature testing

### PR DESCRIPTION
**Pull request by sanjair-git**
### Description of PR

'test_container_autorestart' test case fails for swss, syncd, teamd features with certain error messages for J2C+ cards. These errors are documented in sonic-buildimage issue #[12727](https://github.com/sonic-net/sonic-buildimage/issues/12727). This PR helps to ignore these error messages as part of autorestart container testing.

Summary:
Fixes # (issue)
This PR addresses the following:
 - Ignore expected error messages for auto-restart test case for multi-asic chassis, which is documented as part of sonic-buildimage issue #[12727](https://github.com/sonic-net/sonic-buildimage/issues/12727)
 - The added error messages will be reverted once the messages identified in the issue are fixed.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
Currently for multi-asic chassis, 'test_autorestart_container' test case fails with error messages documented in the sonic-buildimage issue #[12727](https://github.com/sonic-net/sonic-buildimage/issues/12727).
Hence, ignoring these error messages as part of 'test_autorestart_container' test case with this known issue.

#### How did you do it?
- Added expected ERROR messages to the regex list for all types of duthosts.

#### How did you verify/test it?
Ran the tests against a multi-asic line card in a T2 chassis.

### CODE REVIEW

**Review by arlakshm — APPROVED** [review]

File: tests/autorestart/test_container_autorestart.py [hunk 1]
```diff
@@ -142,12 +149,12 @@ def ignore_expected_loganalyzer_exception(duthosts, enum_rand_one_per_hwsku_host
     }
 
     feature = enum_dut_feature
-    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
 
     if loganalyzer:
-        loganalyzer[duthost.hostname].ignore_regex.extend(ignore_regex_dict['common'])
-        if feature in ignore_regex_dict:
-            loganalyzer[duthost.hostname].ignore_regex.extend(ignore_regex_dict[feature])
+        for duthost in duthosts:
```
Comment (arlakshm) [inline tests/autorestart/test_container_autorestart.py:154]: As per my understanding, this change is to ignore logs on other linecards relating to fabric ports going down. Is that correct? 

If that's the case, then can we ignore only those logs on all linecards.

The current code will ignore all logs on all linecards. Restarting container on one linecard should not result in error logs on that other linecards in the chassis (except of fabric ports going down)

**Comment by wenyiz2021** [comment]
@sanjair-git can you please sync this file? I have merged to ignore some similar syslog errors in https://github.com/sonic-net/sonic-mgmt/pull/7437

**Review by wenyiz2021 — CHANGES_REQUESTED** [review]
please sync this file before merging, thanks!

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Passed<br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/autorestart/test_container_autorestart.py:114:121: E501 line too long (138 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:132:121: E501 line too long (126 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:133:121: E501 line too long (125 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:136:42: W605 invalid escape sequence '['<br/>tests/autorestart/test_container_autorestart.py:136:54: W605 invalid escape sequence ']'<br/>tests/autorestart/test_container_autorestart.py:142:121: E501 line too long (140 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:143:121: E501 line too long (138 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:145:121: E501 line too long (145 > 120 characters)<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by sanjair-git** [comment]
> please sync this file before merging, thanks!

@wenyiz2021, Synced the file with latest changes. Please take a look.

File: tests/autorestart/test_container_autorestart.py (hunk 1 above)
Comment (sanjair-git) [reply tests/autorestart/test_container_autorestart.py]: Hi @arlakshm,
Modified the code and verified. With the latest changes, 
-  Logs will be ignored on all cards, if the DUT is a supervisor node.
- Logs will be ignored only on that line card, if the DUT is a line card.

Please take a look.

**Review by wenyiz2021 — APPROVED** [review]

**Review by arlakshm — APPROVED** [review]

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Passed<br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/autorestart/test_container_autorestart.py:114:121: E501 line too long (138 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:132:121: E501 line too long (126 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:133:121: E501 line too long (125 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:136:42: W605 invalid escape sequence '['<br/>tests/autorestart/test_container_autorestart.py:136:54: W605 invalid escape sequence ']'<br/>tests/autorestart/test_container_autorestart.py:142:121: E501 line too long (140 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:143:121: E501 line too long (138 > 120 characters)<br/>tests/autorestart/test_container_autorestart.py:145:121: E501 line too long (145 > 120 characters)<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by abdosi** [comment]
@arlakshm can you please merge this.

**Review by arlakshm — APPROVED** [review]

**Pull request merged**